win_etw_macros/src/lib.rs

```diff
@@ -1170,30 +1219,61 @@ fn parse_event_attributes(
                                         lit: lit.clone(),
                                         attrs: Vec::new(),
                                     });
+                                } else if *path == parse_quote!(id) {
```

Comment:
Btw, when I wrote this code, I didn't know about the `Path::is_ident` method. This can be simplified to:

`... if path.is_ident("id")`

and is probably a lot more efficient.